doc/rst/language/spec/records.rst

```diff
@@ -612,9 +612,8 @@ Calling ``delete`` on a record is illegal.
 Default Comparison Operators
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For records, the compiler will supply default comparison operators if
-they are not supplied by the user. In contrast, the user cannot redefine
-``==`` and ``!=`` for classes. The default comparison operators for a
-record examine the arguments’ fields, while the comparison operators for
-classes check whether the l.h.s. and r.h.s. refer to the same class
-instance or are both ``nil``.
+"Record comparisons have a similar behaviour to tuple comparisons (https://chapel-lang.org/docs/master/language/spec/tuples.html#relational-operators).
```

Comment:
```suggestion
Record comparisons have a similar behavior to :ref:`tuple comparisons <_Tuple_Relational_Operators>`.
```

See `doc/rst/language/spec/tuples.rst` for that label to remove the open link in the doc. See `modules/internal/String.chpl:72` for an instance of these kind of references.

Also let's make it American English (behavior)